pkg/sched/intervals.go

```diff
@@ -314,6 +314,44 @@ func ScheduleIntervalSummary(items []Interval, policyTags *PolicyTags) string {
 	return summary
 }
 
+func ScheduleIntervalsPerDay(items []RetainInterval, policyTags *PolicyTags) uint32 {
+	if len(items) == 0 {
+		return 0
+	}
+	var intervalsPerDay uint32
+	for _, iv := range items {
+		var perDay uint32
+		switch iv.Spec().Freq {
+		case PeriodicType:
+			period := time.Duration(iv.Spec().Period).Minutes()
```

Comment:
period can be zero if Period < 60 seconds.

Reply:
Thanks. Assumed Period will at least a a minute. Fixed now.